frontend/src/uxbox/main/ui/workspace/sidebar/options.cljs

```diff
@@ -45,29 +47,43 @@
      :image [:& image/options {:shape shape}]
      nil)])
 
-(mf/defc shape-options-wrapper
-  [{:keys [shape-id page-id] :as props}]
-  (let [shape-iref (-> (mf/deps shape-id page-id)
+(mf/defc options-toolbox
+  {:wrap [mf/memo]}
+  [{:keys [page selected] :as props}]
+  (let [close #(st/emit! (udw/toggle-layout-flag :element-options))
+        on-change-tab #(st/emit! (udw/set-options-mode %))
+
+        options-mode (mf/deref refs/options-mode)
+
+        selected (mf/deref refs/selected-shapes)
+        shape-id (first selected)
+        page-id (:id page)
+        shape-iref (-> (mf/deps shape-id page-id)
                        (mf/use-memo
                         #(-> (l/in [:objects shape-id])
                              (l/derived refs/workspace-data))))
         shape (mf/deref shape-iref)]
-    [:& shape-options {:shape shape}]))
 
-(mf/defc options-toolbox
-  {:wrap [mf/memo]}
-  [{:keys [page selected] :as props}]
-  (let [close #(st/emit! (udw/toggle-layout-flag :element-options))
-        selected (mf/deref refs/selected-shapes)]
-    [:div.element-options.tool-window
-     ;; [:div.tool-window-bar
-     ;;  [:div.tool-window-icon i/options]
-     ;;  [:span (tr "ds.settings.element-options")]
-     ;;  [:div.tool-window-close {:on-click close} i/close]]
-     [:& align-options]
-     [:div.tool-window-content
-      [:div.element-options
-       (if (= (count selected) 1)
-         [:& shape-options-wrapper {:shape-id (first selected)
-                                    :page-id (:id page)}]
-         [:& page/options {:page page}])]]]))
+    [:div.tool-window
+      ;; [:div.tool-window-bar
+      ;;  [:div.tool-window-icon i/options]
+      ;;  [:span (tr "ds.settings.element-options")]
+      ;;  [:div.tool-window-close {:on-click close} i/close]]
+      [:div.tool-window-content
+        [:& tab-container {:on-change-tab on-change-tab :selected options-mode}
+
+         [:& tab-element
+          {:id :design :title (tr "workspace.options.design")}
+          [:div.element-options
+           [:& align-options]
+           [:divelement-options
```

Comment:
creo que te falta in `.`

Reply:
de hecho sobra el element-options entero :) quitado!